Examples/Run/Reconstruction/CMakeLists.txt

```diff
@@ -32,9 +34,12 @@ target_link_libraries(
     ActsExamplesDetectorGeneric
     ActsExamplesMagneticField
     ActsExamplesIoCsv
-    ActsExamplesIoPerformance)
+    ActsExamplesIoPerformance
+    ActsPluginOnnx)
+
+  install(
+    TARGETS
+    ActsExampleCKFTracks
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+endif()
```

Comment:
This requires the Onnx plugin to run the CKF example, but we need to be able to run this without. My suggestion is to add a preprocessor flag such that the Onnx-related parts in the executable are only used if that flag is set. You can add the `ActsExampleCKFTracks` example unconditionally in the CMake configuration and only add the `ActsPluginOnnx` dependency if enabled.